system-test/spanner.ts

```diff
@@ -1311,15 +1342,27 @@ describe('Spanner', () => {
   });
 
   describe('instanceConfigs', () => {
-    it('should list the available instanceConfigs', done => {
+    it('should have created the instance config', function (done) {
+      if (IS_EMULATOR_ENABLED) {
+        this.skip();
+      }
+      instanceConfig.get((err, metadata) => {
+        assert.ifError(err);
+        assert.strictEqual(metadata!.name, instanceConfig.formattedName_);
+        done();
+      });
+    });
+
+
+    it('should list the available instanceConfigs', function (done) {
       spanner.getInstanceConfigs((err, instanceConfigs) => {
         assert.ifError(err);
         assert(instanceConfigs!.length > 0);
         done();
       });
     });
 
-    it('should list the instanceConfigs in promise mode', done => {
```

Comment:
Is there a specific reason that we are changing this notation?

Reply:
I've changed this notation in order to make it consistent across all tests for instance configs.

Note that if we want to skip a test entirely when using an emulator, accessing `this` inside arrow function(this.skip()) throws an error (TS7041: The containing arrow function captures the global value of 'this'). We changed the notation to a regular function to handle this error.